frontend/components/organiser/wrapped/sections/ShareSection.tsx

```
@@ -0,0 +1,138 @@
+              </>
+            )}
+          </motion.button>
+
+          <Link href="/organiser/dashboard" className="min-w-[240px]">
+            <motion.div
+              whileHover={{ scale: 1.05 }}
+              whileTap={{ scale: 0.95 }}
+              className="h-full px-8 py-4 bg-black text-white rounded-full font-semibold text-lg flex items-center justify-center gap-3 hover:bg-gray-800 transition-colors"
+            >
+              <svg className="w-6 h-6" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                <path
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)